apps/full-site-editing/full-site-editing-plugin/block-patterns/class-block-patterns.php

```diff
@@ -47,6 +47,15 @@ public static function get_instance() {
 	 * @return array Filtered editor settings.
 	 */
 	public function register_patterns( $settings ) {
+		// Register Block Pattern Categories.
+		if ( class_exists( 'WP_Block_Pattern_Categories_Registry' ) ) {
+			register_block_pattern_category( 'blog', array( 'label' => _x( 'Blog', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'call-to-action', array( 'label' => _x( 'Call to Action', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'contact', array( 'label' => _x( 'Contact', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'images', array( 'label' => _x( 'Images', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'menu', array( 'label' => _x( 'Menu', 'Block pattern category', 'gutenberg' ) ) );
+		}
```

Comment:
This is currently not working :confused: Might need to add it to a different hook.